src/main/java/com/coveros/selenified/services/Response.java

```diff
@@ -220,26 +224,27 @@ public Object castObject(Object known, JsonElement unknown) {
      */
     public void assertContains(Map<String, Object> expectedPairs) {
         StringBuilder expectedString = new StringBuilder();
-        Success success = (object == null) ? Success.FAIL : Success.PASS;
+        boolean pass = (object == null) ? false : true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1125)